src/dotnet/APIView/APIViewWeb/Pages/Shared/_Layout.cshtml

```diff
@@ -44,6 +44,9 @@
                             <a asp-action="Logout" asp-controller="Account" class="nav-link">Log Out</a>
                         </li>
                         }
+                        <li class="nav-item">
+                            <a href="https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/APIView/APIViewWeb/README.md" class="nav-link">Help</a>
```

Comment:
@praveenkuttappan when we move the apiview source location we need to remember to update this link.

Reply:
Should we just make this an aka link? It had crossed my mind as I was typing it.